tests/acceptance/bootstrap/SearchContext.php

```diff
@@ -42,6 +42,7 @@ class SearchContext implements Context {
 	 * @When user :user searches for :pattern and limits the results to :limit items using the WebDAV API requesting these properties:
 	 * @When user :user searches for :pattern inside folder :scope using the WebDAV API
 	 * @When user :user searches for :pattern inside folder :scope in space :spaceName using the WebDAV API
+	 * @When user :user searches for :pattern within space :spaceName using the WebDAV API
```

Comment:
in this case, we use space as a scope
```suggestion
	 * @When user :user searches for :pattern inside space :scope using the WebDAV API
```